internal/app/fhir/r5/CodeSystem.go

```diff
@@ -3,27 +3,26 @@ package r5
 import (
 	"fmt"
 
+	"github.com/CDCgov/phinvads-go/internal/database/models/xo"
 	"github.com/google/fhir/go/proto/google/fhir/proto/r5/core/codes_go_proto"
 	datatypespb "github.com/google/fhir/go/proto/google/fhir/proto/r5/core/datatypes_go_proto"
 	r5pb "github.com/google/fhir/go/proto/google/fhir/proto/r5/core/resources/code_system_go_proto"
-	"github.com/CDCgov/phinvads-go/internal/database/models/xo"
 )
 
-func SerializeCodeSystemToFhir(cs *xo.CodeSystem) (*r5pb.CodeSystem, error) {
+func SerializeCodeSystemToFhir(cs *xo.CodeSystem, conceptCount int, concepts []*xo.CodeSystemConcept) (*r5pb.CodeSystem, error) {
 	fhirCS := &r5pb.CodeSystem{
 		Id:           newId(cs.Oid),
-		Status:       &r5pb.CodeSystem_StatusCode{Value: codes_go_proto.PublicationStatusCode_DRAFT},
+		Status:       &r5pb.CodeSystem_StatusCode{Value: codes_go_proto.PublicationStatusCode_ACTIVE},
```

Comment:
I think this is fine for now, but for other resources/future use cases, will we want to be pulling this from the db column rather than hard-coding it?

Reply:
Absolutely, but looking in the database, the "status" column has the same value for every row: "Published", so I think the source data just doesn't have this level of granularity. However, later on down the line, if we for example start doing distributed authoring, we could distribute code systems that are in draft status and not yet published